doc/getting_started/build/build_against_python_env.md

```diff
@@ -26,44 +26,28 @@ Create a new build folder and build DLite against Python 3.9
     workon dlite39
     mkdir build-dlite39
     cd build-dlite39
-    cmake -DCMAKE_INSTALL_PREFIX=$WORKON_HOME/dlite39 -DPython3_FIND_VIRTUALENV=ONLY -DPYTHON_VERSION=3.9 ..
+    cmake -DCMAKE_INSTALL_PREFIX=$VIRTUAL_ENV -DPython3_FIND_VIRTUALENV=ONLY -DPYTHON_VERSION=3.9 ..
     cmake --build .
     cmake --install .
-    ctest
 
 If the `-DPYTHON_VERSION=3.9` option to CMake is not sufficient to get CMake to compile against the correct libraries, you can make a copy of `cmake/build-python3.7.cmake` and configure CMake with
 
     cmake -C ../cmake/build-python3.7.cmake ..
 
+By default, [virtualenv] does not set `LD_LIBRARY_PATH`.
+This will result in errors when running, for example, `dlite-codegen` in the example above.
```

Comment:
Where is this example?

Reply:
Line 35

Reply:
Ok, the main reason I asked was for me to check if the patch worked.